src/Raven.Client/Documents/Conventions/DocumentConventions.cs

```diff
@@ -44,7 +44,8 @@ public class DocumentConventions : Client.Conventions
         internal static readonly DocumentConventions DefaultForServer = new DocumentConventions
         {
             SendApplicationIdentifier = false,
-            MaxContextSizeToKeep = new Size(PlatformDetails.Is32Bits == false ? 8 : 2, SizeUnit.Megabytes)
+            MaxContextSizeToKeep = new Size(PlatformDetails.Is32Bits == false ? 8 : 2, SizeUnit.Megabytes),
+            RequestTimeout = TimeSpan.FromSeconds(15)
```

Comment:
This is a huge change from the previous value, no?

Reply:
Yes, previous value was 12 hours if no timeout was provided.
So each request without timeout would be hang for 12 hours.
This is to prevent cases like firewall dropped the packet and no-one knows about it..

Reply:
Note that this is a server-side only and used for the request executor in few places:
- Replication
- SecretKeyHandler
- RavenETL

Reply:
The question is for what this RequestExecutor is used, because the implication of this is also that you cannot push/receive data for more that 15s. This is why it was changed to 12h in the first place and the idea was to 'cut down' the timeout on those operations that needed it only. We already did that in the past and implications were severe. ETL stopped working, etc. At this point I do not see a way to set it globally to such a low number, needs to be done individually for all needed commands.